src/network/gameclient.cc

```diff
@@ -56,6 +56,11 @@
 #include "wui/interactive_player.h"
 #include "wui/interactive_spectator.h"
 
+struct AddOnsMismatch : WLWarning {
```

Comment:
Usually there is "warning" or "exception" in the name of throwables. 
Otherwise this is pretty straight forward.